src/plugins/image/src/main/js/ui/Dialog.js

```diff
@@ -42,6 +42,11 @@ define(
        */
       function getTypeAheadFieldHtml(callback) {
         var cascadeImageChooserUrl = 'CONTEXT_PATH/imagepopup.act?src=<IMG_SRC>&currentSiteId=' + Utils.getGlobalCascadeVariable().Variables.get('currentSiteId');
+
+        if (Settings.getRestrictToFolderId(editor)) {
```

Comment:
Let's save this to a variable instead of calling it twice. I don't think there's much overhead, but a bit more DRY.